scipy/spatial/transform/rotation.py

```diff
@@ -8,6 +8,94 @@
 AXIS_TO_IND = {'x': 0, 'y': 1, 'z': 2}
 
 
+def _elementary_basis_vector(char):
+    b = np.zeros(3)
+    b[AXIS_TO_IND[char]] = 1
+    return b
+
+
+def _make_euler_from_dcm(dcm, seq, extrinsic=False):
+    # The algorithm assumes intrinsic frame transformations. Their
+    # ELEMENTARY dcms are transpose of what we return.
+    # Adapt the algorithm for our case by
+    # 1. Transposing our dcm representation
+    # 2. Reversing both axis sequence and angles for extrinsic rotations
+
+    seq = seq[::-1] if extrinsic else seq
+
+    if dcm.ndim == 2:
+        dcm = dcm[None, :, :]
+    num_rotations = dcm.shape[0]
+
+    # Step 0
+    # Algorithms dcms are transpose of ours
+    dcm = np.transpose(dcm, (0, 2, 1))
```

Comment:
As the next step I suggest to remove this transposition here. I think we can be smarter than that and show that we can adjust the algorithm accordingly without this simplest "fall back".

My idea is that if `R.T = "decomposition from the paper"`, then `R = transpose of this decomposition` and then we do small adjustments to the current code. I think with available working implementation it should be easy enough. But not until better test coverage.

Reply:
I agree. I will start changing the algorithm once we're satisfied with the test coverage.

Reply:
I think we are approaching this point now.